lib/fog/core/scp.rb

```diff
@@ -1,43 +1,5 @@
 module Fog
   module SCP
-    def self.new(address, username, options = {})
-      if Fog.mocking?
-        Fog::SCP::Mock.new(address, username, options)
-      else
-        Fog::SCP::Real.new(address, username, options)
-      end
-    end
```

Comment:
Thats true when we finish the decoupling. As for now we need to keep it otherwise we will break the library for a ton of users.